ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/post_processors/RaoResultPostProcessor.java

```diff
@@ -7,10 +7,69 @@
 
 package com.farao_community.farao.linear_rao.post_processors;
 
+import com.farao_community.farao.closed_optimisation_rao.post_processors.PstElementResultsPostProcessor;
 import com.farao_community.farao.linear_rao.AbstractPostProcessor;
+import com.farao_community.farao.linear_rao.LinearRaoData;
+import com.farao_community.farao.linear_rao.LinearRaoProblem;
+import com.farao_community.farao.ra_optimisation.PstElementResult;
+import com.farao_community.farao.ra_optimisation.RaoComputationResult;
+import com.farao_community.farao.ra_optimisation.RemedialActionResult;
+import com.google.ortools.linearsolver.MPVariable;
+import com.powsybl.iidm.network.Identifiable;
+import com.powsybl.iidm.network.TwoWindingsTransformer;
+
+import java.util.*;
 
 /**
- * @author Pengbo Wang {@literal <pengbo.wang at rte-international.com>}
+ * @author Viktor Terrier {@literal <viktor.terrier at rte-france.com>}
+ * @author Baptiste Seguinot {@literal <baptiste.seguinot at rte-france.com>}
  */
 public class RaoResultPostProcessor extends AbstractPostProcessor {
+
+    private static final double PST_LIMITS_TOLERANCE = 1e-3;
+
+    @Override
+    public void process(LinearRaoProblem linearRaoProblem, LinearRaoData linearRaoData, RaoComputationResult raoComputationResult) {
+
+        List<RemedialActionResult> remedialActionResults = new ArrayList<>();
+        linearRaoData.getCrac().getRangeActions().forEach(
+            rangeAction -> {
+                String rangeActionId = rangeAction.getId();
+                String rangeActionName = rangeAction.getName();
+                String networkElementId = rangeAction.getNetworkElements().iterator().next().getId();
+
+                double rangeActionVar = getRangeActionVariation(linearRaoProblem.getNegativeRangeActionVariable(rangeActionId, networkElementId), linearRaoProblem.getPositiveRangeActionVariable(rangeActionId, networkElementId));
+
+                if (Math.abs(rangeActionVar) > 0) {
+                    Identifiable pNetworkElement = linearRaoData.getNetwork().getIdentifiable(networkElementId);
+                    if (pNetworkElement instanceof TwoWindingsTransformer) {
+                        TwoWindingsTransformer transformer = (TwoWindingsTransformer) pNetworkElement;
+
+                        double preOptimAngle = transformer.getPhaseTapChanger().getCurrentStep().getAlpha();
+                        int preOptimTap = transformer.getPhaseTapChanger().getTapPosition();
+
+                        double postOptimAngle = preOptimAngle + rangeActionVar;
+                        int postOptimTap = getClosestTapPosition(postOptimAngle, transformer);
+
+                        if (postOptimTap != preOptimTap) {
+                            PstElementResult pstElementResult = new PstElementResult(networkElementId, preOptimAngle, preOptimTap, postOptimAngle, postOptimTap);
```

Comment:
Keeping the non approximated angle and the closest tap may cause confusion

Reply:
you're right, I changed it